ui/cryptomaterial/editor.go

```diff
@@ -161,45 +189,52 @@ func (e Editor) Layout(gtx layout.Context) layout.Dimensions {
 	}
 
 	return layout.UniformInset(e.m2).Layout(gtx, func(gtx C) D {
-		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-			layout.Rigid(func(gtx C) D {
-				return layout.Stack{}.Layout(gtx,
-					layout.Stacked(func(gtx C) D {
-						return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-							layout.Rigid(func(gtx C) D {
-								return e.editorLayout(gtx)
-							}),
-							layout.Rigid(func(gtx C) D {
-								if e.errorLabel.Text != "" {
-									inset := layout.Inset{
-										Top:  e.m2,
-										Left: e.m5,
-									}
-									return inset.Layout(gtx, func(gtx C) D {
-										return e.errorLabel.Layout(gtx)
-									})
+		return e.editorCard.Layout(gtx, func(gtx layout.Context) layout.Dimensions {
+			return e.layout(gtx)
+		})
+
+	})
+}
+
+func (e Editor) layout(gtx C) D {
```

Comment:
Rename this to be more descriptive. How does this `layout` method differ from the other `Layout` method? Devs shouldn't need to look at the code in both methods to figure out the difference; the method names alone should help.